openapi/components/schemas/OrderItem.yaml

```
@@ -79,14 +79,27 @@ properties:
             format: double
             description: Usage limit amount in the currency of the order.
             minimum: 0.01
-  trialEndTime:
-    description: Free trial item date time.
+  servicePeriodStartTime:
+    description: Date and time when the item service period starts.
     readOnly: true
     type:
       - 'string'
       - 'null'
     example: null
     format: date-time
+  servicePeriodEndTime:
+    description: Date and time when the item service period ends.
```

suggestion: to avoid [noun stack](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#noun-strings-or-noun-stacks)
```suggestion
    description: Date and time when the service period of the item ends.
```